plugin/trino-base-jdbc/src/test/java/io/trino/plugin/jdbc/TestJdbcQueryBuilder.java

```diff
@@ -591,6 +595,7 @@ public void testAggregationWithFilter()
                 TEST_TABLE,
                 Optional.of(ImmutableList.of(ImmutableList.of(this.columns.get(2)))),
                 projectedColumns,
+                Map.of("s", "sum(\"col_0\")"),
```

Comment:
Does it make sense to have a test similar to this but where the `tupleDomain` contains a constraint on s? Or is that impossible?

Reply:
Good point. This wouldn't work today (or more precisely -- would depend on whether downstream database is SQL standard compliant or eg MySQL).
I can add some validation.